### ISSUE TITLE
is findOne needed after upsert?

### ISSUE THREAD

**Issue by rjrodger** [senecajs/seneca-mongo-store#95]
does the upsert not return the updated document?

https://github.com/senecajs/seneca-mongo-store/blob/master/mongo-store.js#L230

**Comment by rjrodger** [comment]
this was a question - please reply via comment here

**Comment by lilsweetcaligula** [comment]
@rjrodger No, the call to `findOne` is not required. Both `findOneAndUpdate` and `findOneAndReplace` return the new/updated document if the option `returnOriginal: false` is passed to them.
